frontend/src/redux/store.js

```diff
@@ -0,0 +1,7 @@
+import { createStore } from 'redux';
+
+import userReducer from './reducers/userReducer.reducer';
+
+export default function createAppStore() {
```

Comment:
Seems like this function it's not so necessary, I think we can just call 
```suggestion
const store = createStore(userReducer);
```
from App.js

Reply:
It was to be able to add more states and use "CombinedReducers", but we didn't implement it, so I will correct, thanks